internal/service/project/resource_project_migration_test.go

```diff
@@ -139,3 +141,43 @@ func TestMigProject_withLimits(t *testing.T) {
 		},
 	})
 }
+
+// based on bug report: https://github.com/mongodb/terraform-provider-mongodbatlas/issues/2263
+func TestMigGovProject_regionUsageRestrictionsDefault(t *testing.T) {
+	var (
+		orgID       = os.Getenv("MONGODB_ATLAS_GOV_ORG_ID")
+		projectName = acc.RandomProjectName()
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { acc.PreCheckGovBasic(t) },
+		CheckDestroy: acc.CheckDestroyProjectGov,
+		Steps: []resource.TestStep{
+			{
+				ExternalProviders: acc.ExternalProviders("1.15.3"),
+				Config:            configGovSimple(orgID, projectName),
+				Check: resource.ComposeTestCheckFunc(
+					checkExistsGov(resourceName),
+				),
+			},
+			{
+				ExternalProviders: acc.ExternalProviders("1.16.0"),
+				Config:            configGovSimple(orgID, projectName),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
nit: based on what we were discussing last Thurs, have you done a quick manual check to verify the test is actually failing if you write a wrong check assertion? It would help us make sure the test is actually set-up correctly.

Reply:
Yes, I did it :D Next time, I'll try to check in the test before the fix so it can be seen in the CI too (test added push and then fix push) 🚀